db/db_options_test.cc

```diff
@@ -150,7 +151,8 @@ TEST_F(DBOptionsTest, SetBytesPerSync) {
   // 8388608 = 8 * 1024 * 1024
   ASSERT_OK(dbfull()->SetDBOptions({{"bytes_per_sync", "8388608"}}));
   ASSERT_EQ(8388608, dbfull()->GetDBOptions().bytes_per_sync);
-  for (; i < 40; i++) {
+  // should sync approximately 40KB/8KB ~= 5-10 times
```

Comment:
should be 40MB*2/8M since line 159 is doing full compaction for two tables.

also, how about assert the counter is actually in this range?

Reply:
Ah, you are right, thanks! I missed that earlier. Fixed it now.

Reply:
Added the asserts too, as you suggested.